upki/src/main.rs

```diff
@@ -18,21 +19,33 @@ async fn main() -> Result<(), Report> {
             .init();
     }
 
-    let local = match args.cache_dir {
-        Some(dir) => dir,
-        None => match directories::ProjectDirs::from("dev", "rustls", "upki") {
-            Some(dirs) => dirs.data_local_dir().to_owned(),
-            None => return Err(eyre!("cannot determine home directory")),
-        },
+    let config_path = match args.config_file {
+        Some(path) => path,
+        None => find_config_file().wrap_err("cannot find configuration path")?,
```

Comment:
Would still consider moving this logic into `find_config_file()`, having it take a `specified: Option<PathBuf>` argument?